template/owt-immersive.m4

```diff
@@ -0,0 +1,124 @@
+# Build OWT specific modules
+
+ARG OWTSERVER_REPO=https://github.com/open-webrtc-toolkit/owt-server.git
+ARG OPENH264_MAJOR=1
+ARG OPENH264_MINOR=7
+ARG OPENH264_SOVER=4
+ARG OPENH264_SOURCENAME=v${OPENH264_MAJOR}.${OPENH264_MINOR}.0.tar.gz
+ARG OPENH264_SOURCE=https://github.com/cisco/openh264/archive/v${OPENH264_MAJOR}.${OPENH264_MINOR}.0.tar.gz
+ARG OPENH264_BINARYNAME=libopenh264-${OPENH264_MAJOR}.${OPENH264_MINOR}.0-linux64.${OPENH264_SOVER}.so
+ARG OPENH264_BINARY=https://github.com/cisco/openh264/releases/download/v${OPENH264_MAJOR}.${OPENH264_MINOR}.0/${OPENH264_BINARYNAME}.bz2
+ARG LICODE_COMMIT="8b4692c88f1fc24dedad66b4f40b1f3d804b50ca"
+ARG LICODE_REPO=https://github.com/lynckia/licode.git
+ARG LICODE_PATCH_REPO=https://github.com/open-webrtc-toolkit/owt-server/tree/master/scripts/patches/licode/
+ARG NICE_VER="0.1.4"
+ARG NICE_REPO=http://nice.freedesktop.org/releases/libnice-${NICE_VER}.tar.gz
+ARG SCVP_VER=master
+ARG SCVP_REPO=https://github.com/OpenVisualCloud/Immersive-Video-Sample.git
+ARG WEBRTC_REPO=https://github.com/open-webrtc-toolkit/owt-deps-webrtc.git
+ARG SERVER_PATH=/home/owt-server
+ARG OWT_SDK_REPO=https://github.com/open-webrtc-toolkit/owt-client-javascript.git
+ARG OWT_BRANCH=360-video
+ARG OWT_BRANCH_JS=master
+ARG OWT_BRANCH_JS_COMMIT="d727af2927731ff16214d73f57964a992258636d"
+ARG WEBRTC_COMMIT="c2aa290cfe4f63d5bfbb6540122a5e6bf2783187"
+
+ifelse(index(DOCKER_IMAGE,ubuntu),-1,,dnl
+ARG FDKAAC_LIB=/home/build/usr/local/lib/x86_64-linux-gnu
+RUN apt-get update && apt-get install -y -q --no-install-recommends python libglib2.0-dev libboost-thread-dev libboost-system-dev liblog4cxx-dev
+)dnl
+ifelse(index(DOCKER_IMAGE,centos),-1,,dnl
+ARG FDKAAC_LIB=/home/build/usr/local/lib64
+RUN yum install -y -q python-devel glib2-devel boost-devel log4cxx-devel
+)dnl
+
+# Install 360scvp
+RUN cd /home && \
+    git clone ${SCVP_REPO} Immersive-Video-Sample && \
+    cd Immersive-Video-Sample/src/360SCVP && \
+    git checkout ${SCVP_VER} && \
+    mkdir build && \
+    cd build && \
+    cmake -DCMAKE_INSTALL_PREFIX=/usr/local -DCMAKE_INSTALL_LIBDIR=ifelse(index(DOCKER_IMAGE,ubuntu),-1,lib64,lib/x86_64-linux-gnu) ../ && \
+    make -j && \
+    make install DESTDIR=/home/build && \
+    make install && \
+
+# 1. Clone OWT server source code
+# 2. Clone licode source code and patch
+# 3. Clone webrtc source code and patch
+RUN git config --global user.email "you@example.com" && \
+    git config --global user.name "Your Name" && \
```

Comment:
Why do you need to configure user.email and name? Just cloning does not need such configuration.

Reply:
As we use "git am" to apply local patches, which requires "user.name" and "user.email";
Will use "git apply" to replace of "git am", and remove these two configurations.